code/game/machinery/computer/arcade.dm

```diff
@@ -1027,3 +1027,285 @@
+	name = "NT Recruiter Simulator"
+	desc = "Weed out the good from bad employees and build the perfect manifest to work aboard the station."
+	icon_state = "arcade_recruiter"
+	icon_screen = "nanotrasen"
+	circuit = /obj/item/circuitboard/arcade/recruiter
+	var/candidate_name
+	var/candidate_gender
+	var/age
+	var/datum/species/cand_species
+	var/planet_of_origin
+	var/job_requested
+	var/employment_records
+	/// Current "turn" of the game
+	var/curriculums
+	/// Which unique candidate is he?
+	var/unique_candidate
+
+	var/list/planets = list("Earth", "Mars", "Luna", "Jargon 4", "New Cannan", "Mauna-B", "Ahdomai", "Moghes",
+							"Qerrballak", "Xarxis 5", "Hoorlm", "Aurum", "Boron 2", "Kelune", "Dalstadt")
+	/// Planets with either mispellings or ones that cannot support life
+	var/list/incorrect_planets = list("Eath", "Marks", "Lunao", "Jabon 4", "Old Cannan", "Mauna-P",
+									"Daohmai", "Gomhes", "Zrerrballak", "Xarqis", "Soorlm", "Urum", "Baron 1", "Kelunte", "Daltedt")
+
+	var/list/jobs = list("Assistant", "Clown", "Chef", "Janitor", "Bartender", "Barber", "Botanist", "Explorer", "Quartermaster",
+						"Station Engineer", "Atmospheric Technician", "Medical Doctor", "Coroner","Geneticist", "Chaplain", "Librarian",
+						"Security Officer", "Detective", "Scientist", "Roboticist", "Shaft Miner", "Cargo Technician", "Internal Affairs")
+	/// Jobs that NT stations dont offer/mispelled
+	var/list/incorrect_jobs = list("Syndicate Operative", "Syndicate Researcher", "Veterinary", "Brig Physician",
+								"Pod Pilot", "Cremist", "Cluwne", "Work Safety Inspector", "Musician",
+								"Chauffeur", "Teacher", "Maid", "Plumber", "Trader", "Hobo", "NT CEO",
+								"Mime", "Assitant", "Janittor", "Medical", "Generticist", "Baton Officer",
+								"Detecctive", "Sccientist", "Robocticist", "Cargo Tecchhnician", "Internal Afairs")
+
+	var/list/records = list("Ex-convict, reformed after lengthy rehabilitation, dont normally asks for good salaries", "Charged with three counts of aggravated silliness",
+							"Awarded the medal of service for outstanding work in botany", "Hacked into the Head of Personnel's office to save Ian",
+							"Has proven knownledge of the SOP, but no working experience", "Has worked at Mr Changs",
+							"Spent 8 years as a freelance journalist", "Known as a hero for keeping stations clean while its under attack",
+							"Worked as a bureaucratic for the SolGov", "Worked on the R&D department of the Donk Corporation",
+							"Did work for USSP as an translator", "Took care of Toxins, Xenobiology, Robotics and R&D as a single worker in the Research department",
+							"Served for 4 years as a soldier of the Prospero Order", "Traveled through various systems as an businessman",
+							"Worked as an waiter for one year", "Has previous experience as an cameraman",
+							"Spent years of his life being a janitor at the clown university", "Was given numerous good reviews for delivering cargo requests on time",
```

Comment:
Feel free to do what you want here, I just think clown college being an official thing is very funny
```suggestion
							"Spent years of their life being a janitor at Clown College", "Was given numerous good reviews for delivering cargo requests on time",
```